wp-content/plugins/core/src/Templates/Content/Single/Post.php

```diff
@@ -31,13 +40,18 @@ public function defer_get_content() {
 		return apply_filters( 'the_content', get_the_content() );
 	}
 
-	protected function get_featured_image() {
+	protected function get_featured_image( $attrs = [] ) {
+		$default_attrs = [ 'src_size' => 'full', 'srcset_sizes' => [], 'srcset_sizes_attr' => null ];
+
 		$options = [
-			'wrapper_class' => 'item-single__image',
-			'echo'          => false,
+			'wrapper_class'     => 'item-single__image',
+			'echo'              => false,
+			'srcset_sizes_attr' => '(max-width: 767px) 300px, (min-width: 768px) 800px'
```

Comment:
Fwiw, it's usually rare that you need to utilize this argument, unless you are doing some seriously art directed stuff 😄 Probably not necessary for now.

Reply:
Which one, echo?

Reply:
'srcset_sizes_attr'